packages/cli/src/commands/generate/sdl/sdl.js

```diff
@@ -22,6 +24,26 @@ import { files as serviceFiles } from '../service/service'
 
 const IGNORE_FIELDS_FOR_INPUT = ['id', 'createdAt', 'updatedAt']
 
+const missingIdConsoleMessage = () => {
+  const line1 =
+    chalk.bold.yellow('WARNING') +
+    ': Cannot generate CRUD SDL without an `@id` database column.'
+  const line2 = 'If you are trying to generate for a many-to-many join table '
+  const line3 = "you'll need to update your schema definition to include"
+  const line4 = 'an `@id` column. Read more here: '
+  const line5 = chalk.underline.blue(
+    'https://redwoodjs.com/docs/schema-relations'
+  )
+
+  console.error(
+    boxen(line1 + '\n\n' + line2 + '\n' + line3 + '\n' + line4 + '\n' + line5, {
+      padding: 1,
+      margin: 1,
```

Comment:
```suggestion
      margin: { top: 1, bottom: 3, right: 1, left: 2 },
```
adding bottom margin due to getting cropped